src/Symfony/Component/VarExporter/Tests/LazyGhostTraitTest.php

```diff
@@ -248,6 +248,9 @@ public function testPartialInitialization()
         $this->assertFalse($instance->isLazyObjectInitialized());
         $this->assertSame(123, $instance->public);
         $this->assertFalse($instance->isLazyObjectInitialized());
+        $this->assertTrue($instance->isLazyObjectInitialized(true));
+        $this->assertTrue($instance->isLazyObjectInitialized('public'));
+        $this->assertFalse($instance->isLazyObjectInitialized('publicReadonly'));
```

Comment:
this does not show usage of the method for non public properties.

Reply:
I removed the API to check for a specific property, one should use `\array_key_exists($property, (array) $object);` instead.